packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -593,5 +593,20 @@ export const mutation = mutationType({
         return true
       },
     })
+
+    t.field('setTestingTypeAndReconfigureProject', {
+      description: 'Set the selected testing type, and reconfigure the project',
+      type: Query,
+      args: {
+        testingType: nonNull(arg({ type: TestingTypeEnum })),
+      },
+      resolve: async (source, args, ctx) => {
+        ctx.actions.project.setForceReconfigureProjectByTestingType(args.testingType)
```

Comment:
It looks like `ctx.actions.project.reconfigureProject()` handles setting `setForceReconfigureProjectByTestingType` for us. Not sure we need to do it 2x

Reply:
I updated it to be cleaner!